extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/filters/accesslog/DefaultAccessLogReceiver.java

```diff
@@ -321,21 +298,8 @@ public static Builder builder() {
     }
 
     public static class Builder {
-        private Executor logWriteExecutor;
         private Path outputDirectory;
-        private String logBaseName;
-        private String logNameSuffix;
         private boolean rotate;
-        private LogFileHeaderGenerator logFileHeaderGenerator;
-
-        public Executor getLogWriteExecutor() {
-            return logWriteExecutor;
-        }
-
-        public Builder setLogWriteExecutor(Executor logWriteExecutor) {
```

Comment:
should not change public API of course. It seems unused inside the code. Is it possible that some customers depend on this and we can not see this yet? IDE tells its unused.